packages/package_info_plus/package_info_plus/pubspec.yaml

```diff
@@ -44,5 +45,4 @@ dev_dependencies:
   test: ^1.22.0
 
 environment:
-  sdk: ">=2.18.0 <4.0.0"
-  flutter: ">=3.3.0"
```

Comment:
Why min Flutter is dropped?

Reply:
I should add that back. Sorry. In general it's not needed. But it adds some clarity.

Reply:
I understand that people with Dart 3.x for sure have more or less fresh version of Flutter as well, but better have some clarity for some users.